docs/working_with_masu.rst

```diff
@@ -0,0 +1,90 @@
+===========
+Working with Masu
+===========
+~~~~~
+About
+~~~~~
+
+Nise is a developer tool for generating sample cost and use data for testing purposes. Although nise can generate standalone reports, its most common use case is to create reports to be consumed by `koku's <https://github.com/project-koku/koku>`_ python module masu. This section provides a walkthrough for how to use nise along with masu to generate providers in koku.
+
+OCP Providers
+=============
+In order to create any ocp providers with masu, nise will need to place the csv files in the masu expected location. Masu expects the files to be placed under koku's :code:`testing/pvc_dir/insights_local` `directory <https://github.com/project-koku/koku/tree/master/testing/pvc_dir>`_. If :code:`insights_local` does not exist yet you will have to create it.
+
+Basic OCP provider
+------------------
+**Step One: The Nise Command**
+
+To create a basic OCP provider you will have to provide three key peices of information.
+
+   1. :code:`--ocp-cluster-id $(cluster_id)` <string> Ex. my-cluster_id
+   2. :code:`--insights-upload testing/pvc_dir/insights_local` Masu Expected location
+   3. :code:`--static-report-file $(srf_yaml)` Path to a static report `file <https://github.com/project-koku/nise/blob/master/example_aws_static_data.yml>`_
```

Comment:
In this preamble portion, I think we need to add additional detail around why we're using a static file, what it does for us, and why we can't generate ocp-on-aws data without it.

Reply:
Added this sentence in the ocp-on-aws section: `Which is why we are using report files because the autogentated data option does not have the ability to keep the resouce_id the same across AWS and OCP sample data.`